pkgs/applications/networking/google-drive-ocamlfuse/default.nix

```diff
@@ -22,7 +22,7 @@ buildDunePackage rec {
     hash = "sha256-AWr1tcium70rXFKMTv6xcWxndOJua3UXG8Q04TN1Siw=";
   };
 
-  doCheck = lib.versionOlder ocaml.version "5.0";
+  doCheck = lib.versionAtLeast ocaml.version "5";
```

Comment:
What do we have this condition for? Do we have other ocaml versions in nixpkgs?

Reply:
There are a few versions of OCaml available in `nixpkgs` from 4.0 to 5.3.